grpcclient/lc_client.go

```diff
@@ -48,57 +46,75 @@ import (
 
 // LcClientIf ...
 type LcClientIf interface {
+	VCNSetArtifacts(ctx context.Context, req *schema.VCNArtifactsRequest) (*schema.VCNArtifactsResponse, error)
+	VCNSearchArtifacts(ctx context.Context, req *schema.VCNSearchRequest) (*schema.EntryList, error)
+	VCNGetArtifacts(ctx context.Context, req *schema.VCNArtifactsGetRequest) (*schema.EntryList, error)
+	VCNGetSignature(ctx context.Context, in *schema.VCNGetSignatureRequest, opts ...grpc.CallOption) (*schema.VCNGetSignatureResponse, error)
```

Comment:
I think `Signature` is too ambiguous term (e.g. it is easy to confuse it with signature for proof), maybe we should call it `ClientSignature` ?